Prolangle/wwwroot/js/app.js

```diff
@@ -9,3 +9,7 @@ function getBoundingBox(id) {
         Height: rect.height
     };
 }
+
+async function writeTextToClipboard(text) {
+    await navigator.clipboard.writeText(text);
```

Comment:
I vaguely recall reading that this won't work in all browsers.

Reply:
![image](https://github.com/ricardoboss/Prolangle/assets/6266356/ffc2e7f0-5cfa-4846-8235-9989613bec9b)

It looks like it should work on all, but I can't test this on iOS.

Reply:
Ah, it's probably the "must be called within user gesture event handlers" bit. Which you do seem to do.